packages/ruleset/test/pattern-properties.test.js

```diff
@@ -7,7 +7,7 @@ const { patternProperties } = require('../src/rules');
 const { makeCopy, rootDocument, testRule, severityCodes } = require('./utils');
 
 const rule = patternProperties;
-const ruleId = 'ibm-patternProperties';
+const ruleId = 'ibm-pattern-properties';
```

Comment:
This change doesn't affect the test, but the incorrect rule-id was bugging me :)